modules/third-party/org-eclipse-equinox-http-servlet/bnd.bnd

```diff
@@ -1,2 +1,2 @@
 Bundle-SymbolicName: org.eclipse.equinox.http.servlet
-Bundle-Version: 1.2.2.v20211119-2358.LIFERAY-PATCHED-2
\ No newline at end of file
+Bundle-Version: 1.2.2.v20211118-0920-LIFERAY-PATCHED-1
```

Comment:
@evibudai, I just noticed the version is "-" to append the -LIFERAY-PATCHED-1, others used ".". For example, ../third-party/biz-aQute-bnd-annotation/bnd.bnd.  Please changed "1.2.2.v20211118-0920-LIFERAY-PATCHED-1" for 1.2.2.v20211118-0920.LIFERAY-PATCHED-1.